src/core/components/formContainer/FormContainer.module.css

```
@@ -0,0 +1,34 @@
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.errorMessage {
+  font-weight: 400;
+  font-size: 9.5px;
+  line-height: 100%;
+  letter-spacing: 0%;
+  vertical-align: middle;
+
+  min-height: 20px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)